lib/models/download.js

```diff
@@ -5,7 +5,33 @@ import scopeChecks from 'lib/models/plugins/scopeChecks';
 import * as scopes from 'lib/constants/scopes';
 import uploadSchema from 'lib/models/uploadSchema';
 import addCRUDFunctions from 'lib/models/plugins/addCRUDFunctions';
+import { factory } from 'lib/services/files/storage';
 import _ from 'lodash';
+import { SITE_SETTINGS_ID } from 'lib/constants/siteSettings';
+import logger from 'lib/logger';
+import SiteSettings from './siteSettings';
+import Organisation from './organisation';
+import moment from 'moment';
+
+const getExpireTTL = async (download) => {
+  const organisationId = download.organisation;
+
+  const organisationPromise = Organisation.findById(organisationId);
```

Comment:
Probably easier just to remove these variable since they're unused really.

```js
const [organisation, siteSettings] = await Promise.all([
  Organisation.findById(organisationId),
  SiteSettings.findById(SITE_SETTINGS_ID),
]);
```

Reply:
https://github.com/LearningLocker/learninglocker/commit/74073d73fe9161ea26b34cacbea73158b0c02ccd fixed